altiProfil/install/www/altiprofil/js/altiProfil.js

```diff
@@ -222,7 +222,15 @@ function getProfil(p1,p2){
 }
 
 function initAltiProfil() {
-    var map = lizMap.map;
+    const maxZindex = lizMap.mainLizmap.map.getLayers().getArray().reduce(
```

Comment:
Side question, do you think this function would be useful in LWC core ?
It seems it can be used for PgRouting as well ? (every modules adding a new layer on top)

Reply:
`map.addToolLayer()` :)